docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+
+Given that Faraday's law with an applied current efficiency governs electrochemical conversion, inherently no reaction and mass transfer rate laws are considered. In this yield model, the surface area for electrochemical reaction on each electrode is the only relevant sizing variable required. This is used to validate that the current densities of each are tolerable by the material, and it is typically used as the fixed variable where the corresponding area is calculated as a degree of freedom. With no other sizing variables, effects in increased resistance through the electrolytes with further spaced electrodes and similar size dependent phenomena are not considered in the model.
+
+Degrees of Freedom
+------------------
+For a given electrolyzer design, there are 8 degrees of freedom in addition to the inlet state variables (i.e., temperature, pressure, component flowrates) that should be fixed for the model to be fully specified. In typical design cases the following 8 variables are fixed:
+
+   * membrane current density, :math:`i_{mem}`
+   * anode current density, :math:`i_{ano}`
+   * anode overpotential, :math:`\eta_{ano}`
+   * cathode current density, :math:`i_{cat}`
+   * cathode overpotential, :math:`\eta_{cat}`
+   * current, :math:`I`
+   * current efficiency, :math:`\theta_{current}`
+   * voltage efficiency, :math:`\theta_{voltage}` *or* resistance, :math:`R`
+
+Additionally, the electrolysis reactions at the anode and cathode must be specified with stoichiometry and electrochemical potential (Phillips, 2017). By default, the following stoichiometric variables will be fixed to 0 when the electrolyzer model block is constructed. Therefore, only the nonzero stoichiometry must be additionally specified. As an example for the chlor-alkali process, the following reactions occur with sodium ions permeating from the anolyte to catholyte. The stoichiometry is normalized to 1 mole of electrons as intended in the model framework.
+
+Anode:
+
+    .. math::
+
+        & Cl^-\to{\small\frac{1}{2}}Cl_2+e^-\hspace{4em}E=1.21V \\\\
+
+Cathode:
+
+    .. math::
+
+        & H_2O+e^-\to{\small\frac{1}{2}}H_2+OH^-\hspace{4em}E=-0.99V \\\\
+
+The following variables should then be fixed:
+
+.. code-block::
+
+   # membrane properties
+   m.fs.electrolyzer.membrane_ion_transport_number["Liq", "NA+"].fix(1)
+
```

Comment:
If all ions are given a t_mem, those of anions should be fixed at 0.

Reply:
Yes, all variables not overwritten with a nonzero value are fixed to zero within the build of the model.

Reply:
Make sure this is documented if it hasn't been.